cmd/libs/go2idl/conversion-gen/generators/conversion.go

```diff
@@ -280,8 +280,7 @@ func Packages(context *generator.Context, arguments *args.GeneratorArgs) generat
 			}
 		}
 		if !pkgNeedsGeneration {
-			glog.V(5).Infof("  no viable conversions, not generating for this package")
-			continue
+			glog.Errorf("Package %v has 'k8s:conversion-gen' tags but no viable conversions", i)
```

Comment:
??  How do I repro that at head?

On Mon, Aug 22, 2016 at 6:18 AM, Joe Beda <notifications@github.com> wrote:

> In cmd/libs/go2idl/conversion-gen/generators/conversion.go
> <https://github.com/kubernetes/kubernetes/pull/30787#discussion_r75674739>
> :
>
> > @@ -280,8 +280,7 @@ func Packages(context *generator.Context, arguments *args.GeneratorArgs) generat
> >  			}
> >  		}
> >  		if !pkgNeedsGeneration {
> > -			glog.V(5).Infof("  no viable conversions, not generating for this package")
> > -			continue
> > +			glog.Errorf("Package %v has 'k8s:conversion-gen' tags but no viable conversions", i)
>
> The fact that this wasn't outputting a file was confusing your generated
> files make system. I wasn't able to get to a "no op" make without some sort
> of fix here.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/30787/files/9269652b4fc9760754e0b4e47cc2cb995f9d3e24#r75674739>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AFVgVM2XusCz26zLyZDn5qPbaCQ91KPbks5qiaGdgaJpZM4Jmt7e>
> .
>